### ISSUE TITLE
TopographicProjection is not documented

### ISSUE THREAD

**Issue by skinkie** [NeTEx-CEN/NeTEx#472]

**Comment by nick-knowles** [comment]
Yes TOPOGRAPHICAL PROJECTION was added to  NETEx because when working in fare examples (a)  Often   tariff zones and or administrative zones correspond  existing TOPOGRAPHIC zones and one does not want to have to restate all the coordinates  (b) one needs to be able to specify a COUNTRY for an ADMINISTRATIVE ZONEs  but cannot  do so because  of circular dependency issues ( COUNTRY is not in the core generic  framework so cannot be reference from say ZONE PROJECTION )

A simple ZONE PROJECTION does not allow a ZONE to be projected onto another ZONE (only onto a POINT).

TOPOGRAPHICAL PROJECTION allows any ZONE to be directly mapped to  a TOPOGRAPHICAL PLACE, e.g.  a TARIFF ZONE, ADMINISTRATIVE ZONE can be stated as having the same bounds as another zone. 

I have added Transmodel UML - spec still to do
![image](https://github.com/NeTEx-CEN/NeTEx/assets/1826507/1f6bbe43-e05d-437d-94fe-18f3d9fd8d91)
